src/SentryFullStory.ts

```diff
@@ -55,25 +55,37 @@ class SentryFullStory {
         }
       };
 
-      const self = Sentry.getCurrentHub().getIntegration(SentryFullStory);
-      // Run the integration ONLY when it was installed on the current Hub AND isn't a transaction
-      if (self && event.type !== 'transaction' && this.doesFullStoryExist()) {
+      const getFullStoryUrl = (): string => {
         // getCurrentSessionURL isn't available until after the FullStory script is fully bootstrapped.
         // If an error occurs before getCurrentSessionURL is ready, make a note in Sentry and move on.
         // More on getCurrentSessionURL here: https://help.fullstory.com/develop-js/getcurrentsessionurl
+        try {
+          return FullStory.getCurrentSessionURL(true) || 'Current session URL API not ready'
+        } catch (e) {
+          if (e instanceof Error) {
+            return `Unable to get url: ${e.message}`
+          }
+          return 'Unable to get url'
```

Comment:
Is this check necessary? `e.message` should always exist right, since the param `e` in the `catch` block is always an error?

Reply:
My understanding is you can `throw` [anything in Javascript](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/throw) (it's just discouraged to not throw `Error`).

Reply:
Typescript complains if I try to access `message` without performing a typecheck - I have simplified this code a little though.